README.md

```diff
@@ -44,10 +44,10 @@ More details are available [here](https://github.com/rescript-lang/rescript-comp
 
 See [COPYING](./COPYING) and [COPYING.LESSER](./COPYING.LESSER)
 
-The [`ocaml`](ocaml) directory contains the official [OCaml](https://ocaml.org) compiler (version 4.06.1).
+`vendor/ocaml.tar.gz` contains the official [OCaml](https://ocaml.org) compiler (version 4.14.0).
 Refer to its copyright and license notices for information about its licensing.
 
-The `vendor/ninja.tar.gz` contains the vendored [ninja](https://github.com/ninja-build/ninja).
+`vendor/ninja.tar.gz` contains the vendored [ninja](https://github.com/ninja-build/ninja).
```

Comment:
(Actually vendor/ninja.tar.gz does not exist in the repo itself, but it is part of the npm package.)

Reply:
Anything else to change here or is this done?

Reply:
I'd leave it like this for now.